application/controllers/admin/tokens.php

```diff
@@ -541,17 +541,17 @@ public function editToken($iSurveyId)
 
             echo $from . ',' . $until;
             $aData = array(
-                'firstname' => flattenText(Yii::app()->request->getPost('firstname')),
```

Comment:
Why is flattenText removed?

Reply:
Because it was ONLY for admin … via GUI : not for

- register
- remote control

for example.

Question : did you read the mantis issue ?

Reply:
With this one we flatten for all action (using model …) `return filter_var($attribute, FILTER_SANITIZE_STRING);` not **only** for admin … user via GUI

Reply:
See related : https://bugs.limesurvey.org/view.php?id=16884 …

Reply:
CPDB allow tag too

Reply:
OK :)

Reply:
Can resolve ?